modules/opengl/src/canvasgl.cpp

```diff
@@ -95,7 +96,12 @@ void CanvasGL::renderLayer(size_t idx) {
 bool CanvasGL::ready() {
     if (ready_) {
         return true;
-    } else if (glCheckFramebufferStatus(GL_FRAMEBUFFER) == GL_FRAMEBUFFER_COMPLETE) {
+    } else if (glCheckFramebufferStatus(GL_FRAMEBUFFER) == GL_FRAMEBUFFER_COMPLETE &&
+               // Capabilities might not have been registered yet in (OpenGLQt) module constructor
+               ShaderManager::getPtr()
```

Comment:
if we use the QOpenGLContext::globalShareContext() the OpenGLQtModule then this should never happen?